nautobot/ui/src/components/FiltersPanel.js

```diff
@@ -273,27 +271,24 @@ export const FiltersPanelContent = forwardRef(
         const [lookupType, setLookupType] = useState(null);
         const [lookupValue, setLookupValue] = useState("");
 
-        const [triggerGetFieldLookupChoicesQuery, { data, error, isFetching }] =
-            useLazyGetFieldLookupChoicesQuery();
-
         const lookupFieldOptions = useMemo(
             () =>
-                lookupFields.map(({ key, title }) => ({
-                    label: title,
-                    value: key,
+                Object.keys(lookupFields).map((label) => ({
+                    label: label,
+                    value: label,
                 })),
             [lookupFields]
         );
 
         const fieldLookupChoices = useMemo(
             () =>
-                error || isFetching
-                    ? []
-                    : data?.results?.map?.(({ id, name }) => ({
-                          label: name,
-                          value: id,
-                      })) ?? [],
-            [data, error, isFetching]
+                lookupField
+                    ? lookupFields[lookupField].map(([value, label]) => ({
+                        value: value,
+                        label: label
```

Comment:
If parameter has the same name as the reference to its value (e.g. `{ value: value }`), the assignment can be simplified to a single constant/variable (`{ value }`).

```suggestion
                        value,
                        label
```